includes/class-qm-output.php

```diff
@@ -45,8 +50,8 @@ public function admin_menu( array $menu ) {
 
     public function panel_menu( array $menu ) {
         $ids = array_keys( $menu );
-        $request = array_search( 'qm-request', $ids );
-        $position = $request === false ? count( $menu ) : $request;
+        $request = array_search( 'qm-request', $ids, true );
+        $position = false === $request ? count( $menu ) : $request;
```

Comment:
🤮

Reply:
Well yoda style conditions do indeed reduce hastily made mistakes but I guess this type of error (forgetting one "=") is mostly done by beginners. Personally I changed my coding style to incorporate them as I think that this is a quite clever trick :wink:

Do you think parentheses would reduce your aversion? 😁

Reply:
They are ok 👍🏻 I just don’t like them personally.